airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/test/util/IntegrationTest.kt

```diff
@@ -176,14 +175,19 @@ abstract class IntegrationTest(
         streamStatus: AirbyteStreamStatus? = AirbyteStreamStatus.COMPLETE,
         useFileTransfer: Boolean = false,
     ): List<AirbyteMessage> {
+        val fileTransferProperty =
+            if (useFileTransfer) {
+                mapOf(Property("airbyte.file-transfer.enabled", "USE_FILE_TRANSFER") to "true")
```

Comment:
Nit: should we start building a file or two of constants for these env vars / properties?

Reply:
probably yeah. Curious what you think of this pattern, actually
* micronaut property names are constants, and we inject those properties to a data class https://github.com/airbytehq/airbyte/pull/51618/files#diff-a8bd6ecebede6c4b6750211e7182e95de8d24e4d20f0f37ded3b6571b9e446cc
* and put the mapping between property / env var into testFixtures https://github.com/airbytehq/airbyte/pull/51618/files#diff-8c57a3545940adb9dd75423b33d985cd6051f189fa6b9d1cbbe3f3a74aa900c0

b/c in the long run, connectors should never need to touch env vars at runtime, and maybe don't even need the raw properties?

(unless your goal is to actually put all those env vars into a single file for easy reference?)

Reply:
ah, looks like we do inject the raw `airbyte.file-transfer.enabled` property into a few places :/ I'll pull out a constants file

Reply:
@tryangul thoughts on https://github.com/airbytehq/airbyte/pull/52056/commits/035f1e634fb1328bd84f29e821b671b09f80ce9e?

Reply:
:ship

Reply:
yea, my goal is just to provide a place to keep these organized, so we can see what env vars / props we consume at a glance. Not super opinionated on how that is structured.